map_config/bookstore.rb

```diff
@@ -0,0 +1,5 @@
+@map_res = 0.05
```

Comment:
Is it better to place this file in world repo instead of this repo? Thoughts, @samuelgundry?

Reply:
Yes agreed..

Reply:
I think that's a reasonable standard for the new world repositories to keep to.

Reply:
We would need to advertise(via README?) this standard and information on the plugin configuration files in that case

Reply:
I'm less sure, perhaps. Let's leave it here for now. 

My reason against (for now): The map configs depends on the user, and is application specific. Here, we want a certain resolution so the Robot's navigation stack works expectedly. Another user may want a different resolution and x/y.

Reply:
My concern is how will this scale? This sample app will be limited to specifically supporting these couple of worlds?

Reply:
My concern with adding this as a standard for new worlds is that person adding the new world would have to get onboarded onto how the config gets used into the plugin and eventually map generation and what they think is the best defaul config.. On the other hand, when we support other aws-robotics worlds for SA, we can always update the map configs with other things..

For the case when someone uses this feature for their own world, they are expected to bring their own world and own config